index.html

```diff
@@ -222,12 +227,18 @@
 
 			var $vmToggle = $('#vm');
 			var $web3Toggle = $('#web3');
+			var $web3endpoint = $('#web3Endpoint');
 
 			var executionContext = 'vm';
 			$vmToggle.get(0).checked = true;
 
 			$vmToggle.on('change', executionContextChange );
 			$web3Toggle.on('change', executionContextChange );
+			$web3endpoint.on('change', function(){
+				var endpoint = $('#web3Endpoint').val();
+				web3.setProvider( new web3.providers.HttpProvider( endpoint ) );
```

Comment:
Can we also use ipc provider - in case this website is viewed inside AlethZero or Mist?